src/main/java/seedu/address/model/localcourse/LocalCourse.java

```diff
@@ -79,22 +87,22 @@ public boolean equals(Object other) {
 
         LocalCourse otherLocalCourse = (LocalCourse) other;
         return localCode.equals(otherLocalCourse.localCode)
-                && localName.equals(otherLocalCourse.localName)
-                && localUnit.equals(otherLocalCourse.localUnit);
+            && localName.equals(otherLocalCourse.localName)
+            && localUnit.equals(otherLocalCourse.localUnit);
```

Comment:
Do we need to compare localDescription as well?

Reply:
yes